libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/grid-areas/dh-market-participant-actor-grid-areas.component.html

```diff
@@ -0,0 +1,28 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<mat-selection-list
+  (selectionChange)="onModelChanged()"
+  [(ngModel)]="selectedGridAreas"
```

Comment:
suggestion: instead of using two-way binding to `ngModel` let's split it so:

```suggestion
  (ngModelChange)="onModelChanged($event)"
  [ngModel]="selectedGridAreas"
```

`onModelChanged` will now take an array of selected grid areas.

This change makes sure that data flows one way:
Store -> Component -> ngModel
ngModelChange -> Component -> Store

Reply:
fixed @dzhavat